webpack.config.js

```diff
@@ -0,0 +1,26 @@
+const path = require("path")
+
+const { NODE_ENV = "development" } = process.env
+
+module.exports = {
+    mode: NODE_ENV,
+    devtool: "sourcemap",
+    entry: "./src/index.ts",
+    stats: "errors-only",
+    module: {
+        rules: [
+            {
+                test: /\.tsx?$/,
+                use: "ts-loader",
+                include: /src/,
+            },
+        ],
+    },
+    resolve: {
+        extensions: [".ts"],
```

Comment:
If you were ever to introduce React into the project -- `.tsx` files -- then this would need to be amended to include that extension.